tests/conftest.py

```diff
@@ -60,3 +75,28 @@ def review_set(df_arguments):
 def topic_model():
     """TopicModel instance"""
     return TopicModel()
+
+
+@pytest.fixture(scope="function")
+def dummy_argument_selection():
+    """Dummy argument dataframe."""
+    return pd.DataFrame(
+        {
+            "argument": ["arg1", "arg2", "arg3", "arg4", "arg5"],
+            "argument_id": [4, 7, 13, 29, 33],
+            "score": [1, 2, 2, 3, 3],
+            "coherence": [0.3, 0.5, 0.1, 0.05, 0.9],
+        }
+    )
```

Comment:
Might be nice in addition to have test data that is problematic (strings instead of integers or floats; perhaps NA values, etc), so that you can check how your function behaves in that case.

Reply:
I do agree in general. This is actually related to the typical workflow, which will force the data to look like this. To keep the test set clean, I would prefer to keep it like this for now.